packages/amplication-client/src/Resource/codeGeneratorVersionSettings/Catalog.tsx

```diff
@@ -1,19 +1,17 @@
+import React from 'react';
+import ReactMarkdown from 'react-markdown';
 import { EnumTextStyle, ListItem, Text } from "@amplication/ui/design-system";
 
 type CatalogProps = {
   name: string;
-  changelog: string;
+  changelog: string; 
```

Comment:
@xonx4l as shown in the GitHub action failure, we don't allow trailing whitespaces. Please remove this